src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -104,6 +105,13 @@ public HttpApiHandler(DurableTaskExtension config, ILogger logger)
             int i = path.IndexOf(InstancesControllerSegment, StringComparison.OrdinalIgnoreCase);
             if (i < 0)
             {
+                // Retrive All Status in case of the request URL ends e.g. /instances/
+                if (request.Method == HttpMethod.Get
+                    && path.IndexOf(InstancesControllerSegment.TrimEnd('/'), StringComparison.OrdinalIgnoreCase) >= 0)

```

Comment:
Not sure if this is 100% correct. Does `GET /instances12345` also resolve to this API?  What about `GET /FooBarInstances`? Maybe you can simply do an equality check.

```csharp
if (request.Method == HttpMethod.Get &&
    path.Equals(InstancesControllerSegment.TrimEnd('/'), StringComparison.OrdinalIgnoreCase)
```

Reply:
Sorry about that. I fixed that.

Reply:
I've done that.